app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java

```diff
@@ -161,16 +161,28 @@ protected void onProgressUpdate (Integer... values){
 
     @Override
     protected void onPostExecute(Boolean result) {
+
+        String media_message=media.getDisplayTitle();
+
+        //link to string resources
+        String title_resource = context.getResources().getString(R.string.nominating_for_deletion);
+        String success_resource = context.getResources().getString(R.string.success);
+        String message_resource = context.getResources().getString(R.string.message,media_message);
+        String failed_resource = context.getResources().getString(R.string.failed);
+        String could_not_request_deletion_resource = context.getResources().getString(R.string.could_not_request_deletion);
+
         String message;
-        String title = "Nominating for Deletion";
+        String title = title_resource;
 
         if (result){
-            title += ": Success";
-            message = "Successfully nominated " + media.getDisplayTitle() + " for deletion.";
+
+            title += ":"+success_resource;
+            message = message_resource;
+
         }
         else {
-            title += ": Failed";
-            message = "Could not request deletion.";
+            title += ":"+failed_resource;
```

Comment:
This will not work for right-to-left languages, use `%1$s`/`%2$s`/etc syntax

Reply:
Thanks @nicolas-raoul i have also worked on this.